README.md

```diff
@@ -0,0 +1,41 @@
+
+# Decidim Documentation 
+
+This is the new repository for the [Decidim Documentation website](https://antora.org/):
+
+Is built with [Antora](https://antora.org/), although for this version we're going to try to not have multi-repositories, as it's a hassle to work with. The idea is to have all the documents living on this same repository. 
+
+This project is on active development and it's not on production yet.
+
+For working with it, you can do it several ways: 
+
+## Install locally 
+
+### Docker
+
+You can check this locally and install with Docker. For doing this, you'll need to have Docker and docker-compose installed. 
+
+```bash
+git clone https://github.com/decidim/docs-base
+cd docs-base
+docker-compose up
+cd build/site/decidim/
+xdg-open index.html
+```
+
+### Manually
```

Comment:
That seems better, I'll copy it
I'll add also the node installation, but using nvm, as that's better for working with multiple versions of node (it's similar to rbenv on ruby)

Reply:
It's better to manage ruby versionning, but I would avoid it because it's adding complexity. I think developer should learn about managing software version in other places. Someone who know node will be already carrying about it, but it can be confusing for non node dev.

Reply:
**EDIT:**
I'm wrong in this case and it look's good, nvm is not adding complexity